src/travis2docker/templates/Dockerfile_deployv

```diff
@@ -47,3 +47,6 @@ USER {{ user }}
 ENTRYPOINT /entrypoint.sh
 
 WORKDIR $MAIN_REPO_FULL_PATH
+{% for step in build_extra_steps %}
+{{ step }}
```

Comment:
For commands to be intepreted as such in the DOckerfile, you need to use the RUN:

```suggestion
RUN {{ step }}
```

Same in the other template.

In addition, since this is below:

`USER {{ user }}`

The provided commands are run as the odoo user instead of being run with root

Reply:
I did not include RUN since that would limit the commands that can be executed to just that, although I guess there are very limited if any use cases that will require stuff other than `RUN`, I will make the corrections.